group_vars/preproduction-example/sonarqube/java.yml

```diff
@@ -1,2 +1,2 @@
 ---
-java_apt_package: oracle-java8-installer
+java_deb_apt_package: openjdk-11-jdk
```

Comment:
why use "java_**deb_apt**_package" for variable name? it seem like duplicated meaning.

Reply:
@3d-pro please answer

Reply:
there's 2 method for installing java in `java_deb_method` [here](https://github.com/opsta/ansible-java/blob/master/defaults/main.yml#L2).

Since there's two subset method, that's why the variable will be `java_deb_apt_*` or `java_deb_ppa_*`

I can also rename to just `java_apt_*` or `java_ppa_*` to make it less confusing.

Reply:
From the example in "ansible-java" role, I think you want to install java version 11 or 12 by using custom ppa repository.

I've some idea likes:

Use only one variable to select package name e.g. `java_apt_package` (scope in this variable is Debian) 
because you also install `java_deb_ppa_package` with apt method.

Reply:
After some discussion, we decided to remove ppa method for installing java. I'll update the variables according to the new changes.